init/src/lib.rs

```diff
@@ -37,8 +42,30 @@ impl Args for Options {
         let mut parser = lexopt::Parser::from_args(args);
         let mut path: Option<PathBuf> = None;
 
-        if let Some(arg) = parser.next()? {
+        let mut name = String::from("");
+        let mut description = String::from("");
+        let mut branch = String::from("");
+
+        while let Some(arg) = parser.next()? {
             match arg {
+                Long("name") => {
+                    let value = parser.value()?;
+                    let value = value.to_string_lossy();
+                    let value = value.as_ref();
+                    name = value.to_string();
+                }
+                Long("description") => {
+                    let value = parser.value()?;
+                    let value = value.to_string_lossy();
+                    let value = value.as_ref();
+                    description = value.to_string();
+                }
+                Long("branch") => {
```

Comment:
For all three options, since we don't want users to specify them more than once, we should add a guard like `Long("branch") if branch.is_empty() => {`